tsl/src/data_node.c

```diff
@@ -842,7 +842,7 @@ data_node_attach(PG_FUNCTION_ARGS)
 		ereport(ERROR,
 				(errcode(ERRCODE_INVALID_PARAMETER_VALUE),
```

Comment:
This might be not accurate, since this is likely not related to the parameter value. I wonder if it makes sense to use an existing or add a new errcode related to limits